app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -49,7 +49,7 @@ class PatientComponent extends Component {
 
   searchDemographics(event) {
     event.preventDefault();
-    const { gender, minAge, maxAge, startDate, endDate } = this.state;
+    const { gender, minAge, maxAge, startDate, endDate, livingStatus } = this.state;
```

Comment:
@dkayiwa No, I listed it in the summary. It was one of the glitch we noticed during the demo.